gui/tasks/distribution.js

```diff
@@ -166,12 +166,15 @@ const config = {
     artifactName: 'MullvadVPN-${version}_${arch}.${ext}',
     category: 'Network',
     icon: distAssets('icon.icns'),
-    extraFiles: [{ from: distAssets('linux/mullvad-gui-launcher.sh'), to: '.' }],
+    extraFiles: [
+      { from: distAssets('linux/mullvad-gui-launcher.sh'), to: '.' },
+      { from: distAssets(path.join(getLinuxTargetSubdir(), 'mullvad-daemon')), to: '.' },
```

Comment:
Won't this install `mullvad-daemon` in two places? Further down we install it to `/usr/bin/` also.